openmdao/docs/openmdao_book/features/core_features/adding_desvars_cons_objs/adding_design_variables.ipynb

```diff
@@ -208,6 +208,56 @@
     "assert_near_equal(obj['comp2.y2'][0], 40.555555555555586)\n",
     "assert_near_equal(con['comp1.y1'][0], 21.111111111111143)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# How the optimizer sees scaled variables\n",
+    "\n",
+    "As stated above, when dealing with both a unit conversion and user-defined scaling for design variables, the quantity is first converted to the specified units and then scaled according to the users `ref|ref0|scaler|adder` specification.\n",
```

Comment:
I wonder if we should be clear that the unit conversion here is for when a `units` was given in the add_design_var call, rather than the units that were specified in the add_input of the component.

Reply:
Yeah that makes sense, I'll be more explicit about that.